plugins/panel/plugin.js

```diff
@@ -443,8 +443,14 @@
 						index = this._.focusIndex;
 						focusable = index >= 0 && this._.getItems().getItem( index );
 
-						if ( focusable )
-							focusable.$[ keyAction ] ? focusable.$[ keyAction ]() : focusable.$[ 'on' + keyAction ]();
+						if ( focusable ) {
+							focusable.$[ keyAction ] ? focusable.$[ keyAction ]() :
+							// We must pass info about clicked button (#2857).
+							CKEDITOR.tools.fireElementEventHandler( focusable, 'on' + keyAction, {
+								button: ( CKEDITOR.env.ie && ( CKEDITOR.env.version < 9 || CKEDITOR.env.ie6Compat ) ) ?
+									1 : CKEDITOR.MOUSE_BUTTON_LEFT
```

Comment:
This IE8 magic number which additionally can be mistaken with a middle button click (thus `1 == CKEDITOR.MOUSE_BUTTON_MIDDLE` should be replaced with correct mapping.